src/main/java/me/zeroeightsix/kami/util/graphics/ESPRenderer.kt

```diff
@@ -102,4 +114,10 @@ class ESPRenderer {
     private enum class Type {
         FILLED, OUTLINE, TRACER
     }
+
+    private fun threeDCull(box: AxisAlignedBB): Boolean {
+        val boxvec = box.center.subtract(campos)
+        //fov changes the cone size change for testing
+        return acos(boxvec.dotProduct(lookvec) / (boxvec.length() * lookvec.length())) < (mc.gameSettings.fovSetting/180 * Math.PI)
```

Comment:
Maybe take the display aspect into the calculation so it doesn't cull boxes on sides?